turbo-repo/apps/app/src/lang/en.json

```
@@ -1547,6 +1551,11 @@
       "detectColumns": "Detect columns",
       "addColumn": "Add column",
       "cardLayout": "Card Layout",
+      "horizontal": "Horizontal",
+      "vertical": "Vertical",
+      "chartType": "Chart Type",
```

Biome (2.4.10)

[error] The key chartType was already declared.

(lint/suspicious/noDuplicateObjectKeys)